Dockerfile

```diff
@@ -128,6 +128,15 @@ RUN gcc -DHOMEDIR=\"/data/riotbuild\" -DUSERNAME=\"riotbuild\" /tmp/create_user.
     && chmod u=rws,g=x,o=- /usr/local/bin/create_user \
     && rm /tmp/create_user.c
 
+# Installs the complete ESP8266 toolchain in /opt/esp (170 MB after cleanup) from binaries
+RUN echo 'Adding esp8266 toolchain' >&2 && \
+    cd /opt && \
+    git clone https://github.com/gschorcht/RIOT-Xtensa-ESP8266-toolchain.git esp && \
+    cd esp && \
+    git checkout v1.0
```

Comment:
no, please use the actual commit hash.

Reply:
True, a tag can be replaced at any time, I thought about it afterwards. It was a early ACK.